### DISCUSSION TITLE
hcxdumptool ERROR and WARNING messages

### DISCUSSION THREAD

**Discussion by Zoot420** [ZerBea/hcxdumptool#360] [General]
Anyone know how to fix the error below?

1 ERROR(s) during runtime
Warning: no packets received (frame injection may not work as expected)
Possible reasons:
 no AP in range
 frames are filtered out by BPF
 driver is broken
 driver does not support frame injection

I tried to delete and clone the latest version but it continues to say this.

**Comment by ZerBea** [comment]
Please give some more information:

your command line
`$ sudo hcxdumptool .....`

output of
`$ hcxdumptool -v`

output of
`$ hcxdumptool -L`

**Comment by ZerBea** [comment]
You may have noticed that I have moved this report to discussions, because it is not a hcxdumptool bug.
If hcxdumptool print an ERROR or a WARNING it means that there is something wrong with the command line, the configuration or the driver.
